skp_generator/lib/tests/cupertino_alert_dialog.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart' show FlutterLogo;
+
+class CupertinoDialogTest extends StatelessWidget {
+  CupertinoDialogTest({ Key? key, this.visible = true }) : super(key: key);
+
+  final bool visible;
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: Stack(
+        children: <Widget>[
+          Positioned.fill(
+            child: FlutterLogo(),
+          ),
+          if (visible)
+            Center(
+              child: CupertinoAlertDialog(
+                title: const Text('Dialog Title'),
+                content: const Text(
+                  'This is lots of dummy text. This is lots of dummy text. This is lots of dummy text. '
+                  'This is lots of dummy text. This is lots of dummy text. This is lots of dummy text. '
+                  'This is lots of dummy text. This is lots of dummy text. This is lots of dummy text. '
+                  'This is lots of dummy text. This is lots of dummy text. This is lots of dummy text. '
+                  'This is lots of dummy text. This is lots of dummy text. This is lots of dummy text. ',
```

Comment:
nit:

```suggestion
                  'This is lots of dummy text. This is lots of dummy text. This is lots of dummy text. ' * 5,
```

Reply:
Well if we're going down that road, let's just make it one copy of the sentence `* 15` :-)